apiserver/pkg/server/job_server.go

```diff
@@ -80,14 +80,16 @@ func (s *RayJobServer) ListRayJobs(ctx context.Context, request *api.ListRayJobs
 }
 
 // Finds all Jobs in all namespaces.
-func (s *RayJobServer) ListAllRayJobs(ctx context.Context, _ *api.ListAllRayJobsRequest) (*api.ListAllRayJobsResponse, error) {
-	jobs, err := s.resourceManager.ListAllJobs(ctx)
+func (s *RayJobServer) ListAllRayJobs(ctx context.Context, request *api.ListAllRayJobsRequest) (*api.ListAllRayJobsResponse, error) {
```

Comment:
Can you also add doc strings for the functions `ListAllRayJobs` and `ListJobs`.

Reply:
For grpc services, I prefer to document at protobuf definition file, where we could record usage for fields, requests, responses, and service endpoints.
I think current documentation is not bad: https://github.com/ray-project/kuberay/blob/06367a34b17eeecee6bee51cf4ca7b24ac3412e2/proto/job.proto#L28
Let me know what you think :)

Reply:
both are fine